db/schema.rb

```diff
@@ -83,13 +83,6 @@
     t.index ["user_id"], name: "index_organizers_on_user_id"
   end
 
-  create_table "participants", force: :cascade do |t|
-    t.integer "attendee_id"
-    t.string "attendee_type"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-  end
```

Comment:
This looks like you deleted a table? Once you merged in the dev see if this diff is gone. Otherwise check if migrating again fixes it.